kanidm_unix_int/src/daemon.rs

```diff
@@ -373,6 +373,7 @@ async fn main() {
     if cfg_path.exists() {
         let cfg_meta = metadata(&cfg_path).unwrap_or_else(|e| {
             error!("Unable to read metadata for {} - {:?}", cfg_path_str, e);
+            // Is is safe to call this? Docs say that no destructors will be run...
```

Comment:
At this point it's so early in the lifecycle that killing everything is fine - it hasn't opened the database or anything else. Memory will be cleaned up etc.

Reply:
And even if, we don't system.exit() within a db txn, so we know the ondisk content is stable, so this is fine.